autopart-hibernation.ks.in

```diff
@@ -0,0 +1,70 @@
+#test name: autopart-hibernation
+%ksappend repos/default.ks
+network --bootproto=dhcp
```

Comment:
I've just noticed that the `network` command is already part of `common/common_no_storage_and_payload.ks`, so it can be removed from here.

Reply:
Oh, I missed that, thanks for patience.